20200917_EnumProjectWeek/src/de/telran/data/Schedule.java

```diff
@@ -0,0 +1,30 @@
+package de.telran.data;
+
+public class Schedule {
+    private WeekDays[] days;
+
+    public Schedule(WeekDays[] days) {
+        this.days = days;
+    }
+
+    public void wakeUpWeek() {
```

Comment:
what does this method do? for what ?

Reply:
Fixed. Two last pull requests are correct. That is because I could not find the 3-rd on this page and decided to repeat it, that's why there two pull requests.